Include/internal/pycore_ast.h

```diff
@@ -192,16 +192,7 @@ struct _stmt {
             asdl_expr_seq *decorator_list;
             expr_ty returns;
             string type_comment;
-        } FunctionDef;
-
-        struct {
-            identifier name;
-            arguments_ty args;
-            asdl_stmt_seq *body;
-            asdl_expr_seq *decorator_list;
-            expr_ty returns;
-            string type_comment;
-        } AsyncFunctionDef;
+        } FunctionDef, AsyncFunctionDef;
```

Comment:
Did you just edit a generated file here? (See note at top of file.)

Reply:
Noticed this now.  Dropped the commit from the PR.